src/containers/PageBuilder/Primitives/Image/Image.js

```diff
@@ -41,17 +42,28 @@ export const FieldImage = React.forwardRef((props, ref) => {
   const { width: aspectWidth, height: aspectHeight } = firstImageVariant || {};
 
   const classes = classNames(rootClassName || css.fieldImage, className);
+  const imageLinkHref = props.link?.href || null;
+
+  const responsiveImage = (
+    <ResponsiveImage
+      className={css.fieldImage}
+      ref={ref}
+      alt={alt}
+      image={image}
+      variants={variantNames}
+      sizes={sizes}
+      {...otherProps}
+    />
+  );
   return (
     <AspectRatioWrapper className={classes} width={aspectWidth || 1} height={aspectHeight || 1}>
-      <ResponsiveImage
-        className={css.fieldImage}
-        ref={ref}
-        alt={alt}
-        image={image}
-        variants={variantNames}
-        sizes={sizes}
-        {...otherProps}
-      />
+      {imageLinkHref ? (
+        <Link href={imageLinkHref} title={alt}>
```

Comment:
Your IRL question was a good one! I think, it would be beneficial to include another extra parameter

`<Link href={imageLinkHref} title={alt} fieldType={imageLinkFieldType}>`

It's not used now inside the Link component, but it would be pretty annoying to figure out why the info does not go through in these scenarios with nested components.

Side note: you have already extracted **link** from **props** at this point:
`const imageLinkHref = props.link?.href || null;`